### ISSUE TITLE
Multiple definition of tags on every policy file

### ISSUE THREAD

**Issue by spassarop** [nahsra/antisamy#53]
On every policy XML file, I've found that the tag "col" is defined twice.

The first appearance is:
```xml
<tag name="col" action="validate"/>
```
And the second is:
```xml
<tag name="col" action="validate">
    <attribute name="align" />
    ...
    <attribute name="width" />
</tag>
```
Same thing happens with the property "clip", but this one is and exact copy.

This is not a real issue in Java, as the tests would fail when parsing the policy if it was. However, when implementing this on .NET for example, it may fail if you consider it appears once.

My proposal is to first verify if it is there on purpose. If it isn't, remove it. In my opinion, it makes no sense to have multiple definitions for anything here, it is just confusing for anyone who doesn't know about it as in Java this doesn't fail.

If it is OK to remove the duplicated tags, I can remove them and make the pull request (of course, tests don't fail when removing the tags from the default policy).

**Comment by davewichers** [comment]
@spassarop - Yes. Please do this. I'm going to try to create a new release for AntiSamy this month which bumps some dependencies. So we can slip this change in too.

**Comment by davewichers** [comment]
This is now fixed in the 1.5.11 branch and will be included in the 1.5.11 release, which should come out sometime this month.
